pkg/network/tracer/tracer.go

```diff
@@ -242,8 +243,16 @@ func newConntracker(cfg *config.Config, bpfTelemetry *nettelemetry.EBPFTelemetry
 
 	var c netlink.Conntracker
 	var err error
-	if c, err = NewEBPFConntracker(cfg, bpfTelemetry); err == nil {
-		return c, nil
+
+	cmd := exec.Command("modprobe", "nf_conntrack_netlink")
+	err = cmd.Run()
+
+	if err != nil {
+		log.Warnf("error enabling probe nf_conntrack_netlink: %s", err)
```

Comment:
Line 250 below will also output this err, so maybe we don't need to log here separately?